### ISSUE TITLE
Method that help you create .jsonl

### ISSUE THREAD

**Issue by Errahum** [Errahum/HeliosTuner-OpenAI-fine-tuning#3]
1. Backend --> Jsonl creator with inputs for the users. There will be three inputs:
- [x] First will be for the system role.
- [x] Second is for the user role.
- [x] Third is for the assistant role.

3. Logic:
- [x] The user will be able to name the jsonL file. 
- [x] If a jsonL file already exist with the same name, each new line in the jsonL will be added.
- [x] There must be a converter for the code to be included in the jsonL. In addition, special characters must be compatible.

5. Frontend --> thinker
- [x]  Input with label GUI
- [x] Save function that use the input value.
- [x] Clear field function that clear all the input except system.

6. Add a button on the main interface of the finetuning GUI.
